weldx/welding/groove/iso_9692_1.py

```diff
@@ -240,19 +240,22 @@ def to_profile(self, width_default: pint.Quantity = Q_(5, "mm")) -> geo.Profile:
              pint.Quantity (Default value = Q_(5, "mm"))
 
         """
-        t = self.t.to(_DEFAULT_LEN_UNIT).magnitude
-        b = self.b.to(_DEFAULT_LEN_UNIT).magnitude
-        width = width_default.to(_DEFAULT_LEN_UNIT).magnitude
+        t = self.t
+        b = self.b
+        width = width_default
 
         # x-values
-        x_value = [-width, 0, 0, -width]
+        x_value = np.stack((-width, 0, 0, -width))
+
         # y-values
-        y_value = [0, 0, t, t]
+        y_value = Q_([0, 0], "mm")
+        y_value = np.append(y_value, t)
+        y_value = np.append(y_value, t)
```

Comment:
this should work with `np.stack`
```suggestion
        y_value = np.stack((0,0,t,t))
```

please check the rest of the code for similar cases